src/inputProcessor.js

```diff
@@ -38,8 +38,8 @@ function getSource(svg) {
   const result = {
     top: rect.top,
     left: rect.left,
-    width: rect.width,
-    height: rect.height,
+    width: svg.width.baseVal.value,
```

Comment:
Apparently, it's a bit more complicated than I thought. Chrome and Firefox approach this value a bit differently. I'm trying to set up some tests as it feels that solution gonna be a bit more complicated.

Reply:
Yes, agreed. I do have a bit of time to help investigate this in the next few weeks, but that's also why I'm proposing that I break out commit 432f198 into a separate PR, since it's a different issue. What do you think?

Reply:
Yeah, let’s do it!